docs/cookbook/basics.md

```diff
@@ -0,0 +1,193 @@
+# Basics
+
+!!! tip
+    This page is up-to-date with version `1.0.0-beta.1` of `bdk_wallet`.
+
+Add BDK
+
+=== "Rust"
+
+    ``` toml
+    [dependencies]
+    bdk_wallet = { version = "1.0.0-beta.1", features = ["keys-bip39"] }
+    bdk_esplora = {  version = "0.16.0" , features = ["blocking"] }
+    ```
+
+=== "Swift"
+
+    1. From the Xcode File menu, select Add Package Dependencies...
+    2. Enter `https://github.com/bitcoindevkit/bdk-swift` into the package repository URL text field
+
+=== "Kotlin"
+
+    ``` kotlin
+    repositories {
+        mavenCentral()
+    }
+
+    dependencies {
+    
+        // for jvm
+        implementation 'org.bitcoindevkit:bdk-jvm:<version>'
```

Comment:
Didn't add a specific version for Kotlin (or Swift) because there is no bindings version yet, will follow up later once we have one